Chap_API_Query.tex

```diff
@@ -611,21 +211,21 @@ \subsection{\code{PMIx_Query_info_nb}}
 \pasteAttributeItemEnd
 \pasteAttributeItemBegin{PMIX_RANK}Specifies the rank of the process whose information is being requested. Must be accompanied by the \refattr{PMIX_NSPACE} attribute. Only required when the request is for information on a specific process.
 \pasteAttributeItemEnd
-\pasteAttributeItem{PMIX_QUERY_ATTRIBUTE_SUPPORT}
 \pasteAttributeItem{PMIX_CLIENT_ATTRIBUTES}
 \pasteAttributeItem{PMIX_SERVER_ATTRIBUTES}
 \pasteAttributeItem{PMIX_HOST_ATTRIBUTES}
 \pasteAttributeItem{PMIX_TOOL_ATTRIBUTES}
 
-Note that inclusion of both the \refattr{PMIX_PROCID} directive and either the \refattr{PMIX_NSPACE} or the \refattr{PMIX_RANK} attribute will return a \refconst{PMIX_ERR_BAD_PARAM} result, and that the inclusion of a process identifier must apply to all keys in that \refstruct{pmix_query_t}. Queries for information on multiple specific processes therefore requires submitting multiple \refstruct{pmix_query_t} structures, each referencing one process.
-
-\ac{PMIx} libraries are not required to directly support any other attributes for this function. However, all provided attributes must be passed to the host \ac{SMS} daemon for processing. The \ac{PMIx} library is \textit{required} to add the \refAttributeItem{PMIX_USERID} and the \refAttributeItem{PMIX_GRPID} attributes of the client process making the request.
+Note that inclusion of both the \refattr{PMIX_PROCID} directive and either the \refattr{PMIX_NSPACE} or the \refattr{PMIX_RANK} attribute will return a \refconst{PMIX_ERR_BAD_PARAM} result, and that the inclusion of a process identifier must apply to all keys in that \refstruct{pmix_query_t}. Queries for information on multiple specific processes therefore requires submitting multiple \refstruct{pmix_query_t} structures, each referencing one process.  Directives which are not applicable to a key are ignored. 
 
+% Use of pmix_server_query_fn is covered in server interfaces chapter
 \reqattrend
 
 \optattrstart
```

Comment:
This was supposed to be removed as it was for the blocking API presentation.  These are not optional attributes, these are keys.  The blocking case is correct, see that one.